util/util.go

```diff
@@ -182,8 +225,18 @@ func Marshal(data interface{}) (v starlark.Value, err error) {
 			}
 		}
 		v = dict
+	case Marshaler:
+		v, err = x.MarshalStarlark()
 	default:
 		return starlark.None, fmt.Errorf("unrecognized type: %#v", x)
 	}
 	return
 }
+
+type Unmarshaler interface {
+	UnmarshalStarlark(starlark.Value) error
+}
```

Comment:
This will cause `golint` to fail. Both the `Unmarshaler` interface and `UnmarshalStarlark` methods should have comments that explain their use.